src/api/Metadata.js

```diff
@@ -354,22 +352,118 @@ class Metadata extends File {
         return editors;
     }
 
+    /**
+     * Utility to concat instance and template into one entity.
+     *
+     * @param {Object} instance - metadata instance
+     * @param {Object} template - metadata template
+     * @return {Object} metadata template instance
+     */
+    createTemplateInstance(instance: MetadataInstanceV2, template: MetadataTemplate): MetadataTemplateInstance {
+        const metadataFields: MetadataInstanceTemplateField[] = [];
+
+        // templateKey is unique identifier for the template,
+        // but its value is set to 'properties' if instance was created using Custom Metadata option instead of template
+        const isCustomMetadataInstance = template.templateKey !== METADATA_TEMPLATE_PROPERTIES;
+        if (isCustomMetadataInstance) {
+            // Get Metadata Fields for Instances created from predefinied template
+            const templateFields = template.fields || [];
+            templateFields.forEach(async field => {
+                metadataFields.push({
+                    description: field.description,
+                    displayName: field.displayName,
+                    hidden: field.hidden,
+                    id: field.id,
+                    key: field.key,
+                    options: field.options,
+                    type: field.type,
+                    value: instance[field.key],
+                });
+            });
+        } else {
+            // Get Metadata Fields for Custom Instances
+            Object.keys(instance).forEach(key => {
+                if (!key.startsWith('$')) {
+                    // $FlowFixMe
```

Comment:
Can it be fixed within this PR?

Reply:
This is comment copied from somewhere else. I deleted it.

Reply:
Yes!